src/view/com/feeds/FeedPage.tsx

```diff
@@ -59,6 +60,13 @@ export function FeedPage({
   const feedFeedback = useFeedFeedback(feed, hasSession)
   const scrollElRef = React.useRef<ListMethods>(null)
   const [hasNew, setHasNew] = React.useState(false)
+  const setHomeBadge = useSetHomeBadge()
+
+  React.useEffect(() => {
+    if (isPageFocused) {
+      setHomeBadge(hasNew)
+    }
+  }, [isPageFocused, hasNew, setHomeBadge])
```

Comment:
The goal here is to react to both `isPageFocused` and `hasNew` so making it an effect is the easiest way to do that. We might want to refactor polling more generally and move some of this state around but I'm hesitant to do this now.